src/main/java/core/mate/academy/service/MachineServiceImpl.java

```diff
@@ -1,7 +1,29 @@
 package core.mate.academy.service;
 
+import core.mate.academy.model.Machine;
+
+import java.util.List;
+
 /**
  * Your implementation of MachineService.
  */
-public class MachineServiceImpl {
+public class MachineServiceImpl implements MachineService{
+    @Override
+    public List<? extends Machine> getAll(List <? extends Machine> machines) {
+        return machines;
+    }
```

Comment:
Code quality issue: The `getAll()` method should not just return the input list. Consider creating a local variable in your implementation of `getAll(Class type)` method. For example: `List<? extends Machine> machinesList = someProducer.get(); return new ArrayList<>(machinesList);`